README.md

```diff
@@ -6,15 +6,25 @@ Learn to balance baby.
 
 # Roadmap
 
-## Clean up current code
+Rename this repo. Candidates:
+- Talos
+- PRL
+- IRL (Inference for RL)
 
+## Clean up current code
 - Docstrings - Sphinx
+- Define documentation layout
 - Remove all hacky stuff, like hard coded tensors
-- Separate learning-dynamics, objective, optimisation, plotting
+- Clean up pendulum.py: learning-dynamics, objective, optimisation, plotting
+
+## Example notebooks
+- Pendulum notebook
 
 ## Write derivations, including complexity
+- Moment matching
 
 ## Profiling
+- Find a profiler
```

Comment:
This is not specific enough, please be a bit more specific here.

Reply:
Will make change.